src/technologies/a.json

```diff
@@ -2454,21 +2454,53 @@
     "scriptSrc": "cf\\.aliyun\\.com/",
     "website": "https://help.aliyun.com/document_detail/193141.html"
   },
-  "All in One SEO Pack": {
+  "All in One SEO": {
     "cats": [
       54,
       87
     ],
     "cpe": "cpe:2.3:a:aioseo:all_in_one_seo:*:*:*:*:*:wordpress:*:*",
-    "description": "All in One SEO plugin optimizes WordPress website and its content for search engines.",
-    "html": "<!-- All in One SEO Pack ([\\d.]+) \\;version:\\1",
+    "description": "All in One SEO optimizes a WordPress website and its content for search engines.",
+    "meta": {
+      "generator": "^All in One SEO \\(AIOSEO\\) ([\\d\\.]+)\\;version:\\1"
+    },
+    "dom": {
+      "script.aioseo-schema": {
+        "attributes": {
+          "class": ""
+        }
+      }
+    },
+    "html": [
+      "<!-- All in One SEO Pack ([\\d.]+) \\;version:\\1",
+      "<!-- All in One SEO ([\\d.]+) \\;version:\\1"
+    ],
     "icon": "AIOSEO.svg",
     "pricing": [
       "freemium",
       "low",
       "recurring"
     ],
-    "requires": "WordPress",
+    "implies": "WordPress",
+    "website": "https://aioseo.com"
+  },
+  "All in One SEO Pro": {
+    "cats": [
+      54,
+      87
+    ],
+    "description": "All in One SEO optimizes a WordPress website and its content for search engines.",
+    "html": "<!-- All in One SEO Pro ([\\d.]+) \\;version:\\1",
+    "meta": {
+      "generator": "^All in One SEO Pro \\(AIOSEO\\) ([\\d\\.]+)\\;version:\\1"
+    },
+     "icon": "AIOSEO.svg",
+    "pricing": [
+      "freemium",
+      "low",
+      "recurring"
+    ],
```

Comment:
If this is the "Pro" option then are there freemium and low options? Not that we currentyl use this info, but still...

Reply:
Also taken from how Yoast SEO is setup, but happy to change if I know what to change it to :P

Reply:
Changed it to how it's set for Yoast SEO entirely: `oss` to `true` for the free, and then the pricing thing on the premium thing.